src/settings.rs

```diff
@@ -545,7 +543,7 @@ pub struct NoOpStore;
 
 impl Store for NoOpStore {
     fn load(&self) -> Settings {
-        Default::default()
+        Settings::default()
```

Comment:
I actually like the `Default::default` here a bit more. The `Settings` type is named just a line above and this is a trivial function. I don't like repeating the `Settings` or whatever thing multiple times if I can avoid it.